packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -424,15 +467,25 @@ abstract contract MajorityVotingBase is
 
         if (_isProposalOpen(proposal_)) {
             // Early execution
-            return
-                proposal_.parameters.votingMode == VotingMode.EarlyExecution &&
-                worstCaseSupport(_proposalId) > proposal_.parameters.supportThreshold &&
-                participation(_proposalId) >= proposal_.parameters.minParticipation;
+            if (proposal_.parameters.votingMode != VotingMode.EarlyExecution) {
+                return false;
+            }
+            if (!isSupportThresholdReachedEarly(_proposalId)) {
+                return false;
+            }
+            if (!isMinParticipationReached(_proposalId)) {
+                return false;
+            }
+        } else {
+            // Normal execution
+            if (!isSupportThresholdReached(_proposalId)) {
+                return false;
+            }
+            if (!isMinParticipationReached(_proposalId)) {
```

Comment:
move `isMinParticipationReached` outside of the `if`. The calls in the `if` and `else` are the same.

Reply:
Fixed in c12d89e55b76ae444c19f88eeadeaeff0f1825b8